src/scripts/main/public/styles/code-log.css

```diff
@@ -242,8 +242,123 @@ button.frame {
   border-radius: 5px 5px 0 0;
   opacity: 1;
 }
+/* Styles for log entries and container */
+.logs-container {
+  max-height: 100%; /* Adjust this value as needed */
+  overflow-y: scroll;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  padding: 10px;
+}
+
+#logsContainer {
+    height: 100%;
+    overflow-y: auto;
+    padding-bottom: 100px;
+    
+}
+
+.log-entry {
+    background: #ffffff;
+    margin-bottom: 20px;
+    width: 250px;
+    height: 250px;
+    border-radius: 8px;
+    display: flex;
+    flex-direction: column;
+}
+
+.fieldD1,
+.fieldD2 {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    padding: 10px;
+    width: 100%;
+    border-radius: 8px;
+}
+
+.delete-button {
+    top: 0;
+    right: 0;
+    background: none;
+    border: none;
+    font-size: 1.15em;
+    cursor: pointer;
+    color: red;
+  }
+
+.fieldD1 {
+    background: linear-gradient(281deg, rgba(253, 185, 39, 0.60) 20.93%, #FDB927 82.58%);
+}
+
+.fieldD2 {
+    background: white;
+    border: 2px solid gold;
+    overflow-y: scroll; /* Enable vertical scrolling */
+    height: 150px; /* Set a fixed height for consistency */
+}
+
+.date-codelog,
+.time-codelog {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    display: block;
+    color: white;
+    font-size: 20px;
+}
+
+.hidden {
+    display: none;
+}
+
+.placeholder-text {
+    padding: 2px;
+    color: grey;
+    font-style: italic;
+}
+
+.log-entry p {
+  margin: 5px 0;
+  line-height: 1.5;
+}
+
+.log-entry strong {
+  color: #552583;
+  font-size: 16px;
+}
+
+/* Responsive Styles */
+@media (max-width: 768px) {
+    .container {
+        flex-direction: column;
+        margin: 10px;
+        width: calc(100% - 20px);
+    }
+
+    .feedbacklist {
+        width: 100%;
+        max-width: none;
+    }
+
+    .modal-content {
+        width: 90%;
+        height: auto;
+    }
+
+    .log-entry {
+        width: calc(100% - 20px); /* Full width for mobile view */
+    }
+
+    .fieldD2 {
+        height: auto; /* Adjust height for mobile view */
+        overflow-y: scroll;
+    }
+}
```

Comment:
I think you are trying to adapt the Responsive CSS from the Feedback Page, could you change the HTML identifier to the correct corresponding CodeLog HTML identifiers, so that it responds correctly to mobile screen sizes.

Can you also fix the add button on Mobile devices? (You can add a percentage amount of padding to the button, can look at `feedback.css`

This is what is currently happening:
![image](https://github.com/cse110-sp24-group8/cse110-sp24-group8/assets/122563165/83c7c925-40dc-4650-8e35-9701bed411da)